concepts/index.md

```diff
@@ -4,34 +4,34 @@ layout: documentation
 
```

Comment:
This page is copied over during the Maven build - it must not be manually edited (see https://github.com/openhab/openhab-docs#automatically-generated-parts).

What you can do is to create an openHAB version of it, put it in some other folder and update the menu to point to your version.